scripts/detect_artefacts.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+
+import os
+
+os.environ['USE_TORCH'] = '1'
+
+import argparse
+import logging
+
+import cv2
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import figure
+import torch
+
+from doctr.models import obj_detection
+from doctr.io.image import read_img_as_tensor
+
+
+def plot_predictions(image, tg_boxes, tg_labels, cl_map, cm):
+    for ind_2, val_2 in enumerate(tg_boxes):
+        # Bounding box around artefacts
+        cv2.rectangle(image, (val_2[0], val_2[1]), (val_2[2], val_2[3]),
+                      cm[tg_labels[ind_2]], 2)
+        text_size, _ = cv2.getTextSize(cl_map[int(tg_labels[ind_2])], cv2.FONT_HERSHEY_SIMPLEX, 2, 2)
+        text_w, text_h = text_size
+        # Filled rectangle above bounding box
+        cv2.rectangle(image, (val_2[0], val_2[1]), (val_2[0] + text_w, val_2[1] - text_h),
+                      cm[tg_labels[ind_2]], -1)
+        # Text bearing the name of the artefact detected
+        cv2.putText(image, cl_map[int(tg_labels[ind_2])], (int(val_2[0]), int(val_2[1])),
+                    cv2.FONT_HERSHEY_SIMPLEX, 2, (255, 255, 255), 3)
+    figure(figsize=(9, 7), dpi=100)
```

Comment:
not sure but do we really need to specify the figure here?

Reply:
That was to increase the size of image plot because the default one may be a bit less. I am removing it for the moment though :)